test/certs/embeddedSCTs1_issuer.pem

```diff
@@ -0,0 +1,61 @@
+Certificate:
```

Comment:
perhaps remove the text part here since we don't have it in our other pem files.